### ISSUE TITLE
When using Chrome/Firefox to edit the "Body" text editing box on the Android phone, it is almost impossible to enter any text.

### ISSUE THREAD

**Issue by Songyon** [decaporg/decap-cms#6300]
**Describe the bug**
When using Chrome/Firefox to edit the "Body" text editing box on the Android phone, it is almost impossible to enter any text.

**To Reproduce**
1. You need to have an Android phone.
2. Open the Netlify CMS
3. Click the "Body" textfield and try to input any characters.
4. See error.

**Expected behavior**
Input any characters normally.

**Applicable Versions:**

 - Netlify CMS version: 2.0.10
 - Git provider: GitHub
 - OS: MIUI 13 (Android 12)
 - Browser version: Chrome Dev 100 and Firefox Beta 98

**CMS configuration**
```yaml
backend:
  name: git-gateway # https://github.com/netlify/netlify-cms
  branch: main # 要更新的分支(可选；默认为主分支)

# This line should *not* be indented
publish_mode: editorial_workflow

# This line should *not* be indented
media_folder: "themes/fluid/source/img" # 媒体文件将存储在图片/上载下的Repo中。
public_folder: "/img" # 上传的媒体的src属性将以/images/uploads开头。

site_url:  # 网站网址
display_url:  # 显示网址
logo_url: 

locale: "zh_Hans" # 语言环境 https://github.com/netlify/netlify-cms/tree/master/packages/netlify-cms-locales/src

auto_generator:
  post: 
    # 如果你有多个Post文件夹，在这里定义多个，见https://github.com/jiangtj/blog/blob/master/netlify-cms.yaml
    all_posts:
      # 设置为false，关闭默认的Post
      enabled: false
      label: "Post"
      folder: "source/_posts"
      create: true
      editor:
        preview: true
  # Page生成配置
  page: 
    enabled: false
    config:
      label: "Page"
      # 默认禁止删除Page文件
      delete: false
      editor:
        preview: true

collections:      # https://www.netlifycms.org/docs/configuration-options/#collections
  - name: "posts" # 在路由中使用，例如：/admin/collections/blog。
    label: "文章" # 在用户界面中使用
    
    folder: "source/_posts" # 存储文件的文件夹的路径。
    create: true # 允许用户在这个集合中创建新的文件。
    fields: # 每份文件的字段，通常是前面的内容。
      - {label: "标题", name: "title", widget: "string"}
      - {label: "作者", name: "author", widget: "string", required: false} 
      - {label: "摘要", name: "excerpt", widget: "list", required: false}
      - {label: "文章页顶部大图", name: "banner_img", widget: "image", required: false} 
      - {label: "文章在首页的封面图", name: "index_img", widget: "image", required: false} 
      - {label: "发布日期", name: "date", widget: "datetime", format: "YYYY-MM-DD HH:mm:ss", dateFormat: "YYYY-MM-DD", timeFormat: "HH:mm:ss", required: false}
      # - {label: "更新日期", name: "updated", widget: "datetime", format: "YYYY-MM-DD HH:mm:ss", dateFormat: "YYYY-MM-DD", timeFormat: "HH:mm:ss", required: false}
      - {label: "永久链接", name: "permalink", widget: "string", required: false}
      - {label: "内容", name: "body", widget: "markdown", required: false}
      - {label: "标签", name: "tags", widget: "list", required: false}
      #- {label: "分类", name: "categories", widget: "list", required: false}
      - {label: "文章排序权重", name: "sticky", widget: "number", required: false}
      - {label: "在首页中隐藏此文章", name: "hide", widget: "boolean", default: false, required: false}
      #  comment: 'This is a multiline\ncomment'
      # - {label: "评论区", name: "comments", widget: "boolean", default: true, required: false}
      # - {label: "关键词", name: "keywords", widget: "list", required: false}
    editor:
      preview: true
```

**Comment by 0x3st** [comment]
I’ve the same problem on my ipad.
OS: ipados 15.5
Browser: Safari

**Comment by DRuggeri** [comment]
FYI, this is also reproducible on the Netlify demo page.

To add a little color:
- Text cannot be added with the rich text editor until one of the buttons on the editor is clicked. This action causes the Android keyboard to open.
- Once text is entered, it doesn't seem to be fed into the app. The text is not shown in the preview on the right
- Switching chrome to desktop mode does not help. Im fact, it somehow causes text entered to be prepended (typing "hello" will actually add "olleh" to the text box)
- Text entered is also apparently not accessible when posting the article. The UI gives an error about missing body content

I suspect the issue ties to the WYSIWYG editor since other fields that are plain old text (Title, for example) work fine and accept the textual input. 


![Screenshot_20220905_102911](https://user-images.githubusercontent.com/915761/188481889-838d2664-8ea1-4e5f-873e-34a542b7d232.jpg)

**Comment by timpaul** [comment]
I'm having a similar issue. In the editor, when I try to type text into the middle of an existing line, the cursor automatically moves to the end of that line after each keystroke. 

This happens on both the markdown and rich text view in Chrome (but not in Safari).

**Comment by jeebb** [comment]
The issue is still there in Decap CMS.

**Comment by martinjagodic** [comment]
We are actively working on updating Slate, which is the root of all editor problems. In the meantime, you can try this fix: https://github.com/decaporg/decap-cms/issues/5092#issuecomment-1256321540

**Comment by jeebb** [comment]
It works on my side. Thanks @martinjagodic

**Comment by Songyon** [comment]
> We are actively working on updating Slate, which is the root of all editor problems. In the meantime, you can try this fix: https://github.com/decaporg/decap-cms/issues/5092#issuecomment-1256321540

One year ago😂😂😂

I just realized that this project has been renamed😭😭

**Comment by maxwedwards** [comment]
> We are actively working on updating Slate, which is the root of all editor problems. In the meantime, you can try this fix: [#5092 (comment)](https://github.com/decaporg/decap-cms/issues/5092#issuecomment-1256321540)

This fixed my cursor jumping on chrome on macos but I still can't edit rich text on chrome/firefox on my android phone.

**Comment by martinjagodic** [comment]
@Songyon, @zoran-cuckovic, @jeebb, @timpaul is this issue solved for you in Decap 3.0?

**Comment by Songyon** [comment]
> @Songyon, @zoran-cuckovic, @jeebb, @timpaul is this issue solved for you in Decap 3.0?

Thank you. After my attempt, I confirmed that the problem has been fixed.

**Comment by martinjagodic** [comment]
Great! Closing as fixed.
